__tests__/app.test.js

```diff
@@ -241,10 +241,10 @@ describe("ENDPOINT TESTING", () => {
               title: "UNCOVERED: catspiracy to bring down democracy",
               topic: "cats",
               author: "rogersop",
-              body: "Bastet walks amongst us, and the cats are taking arms!",
               created_at: "2020-08-03T13:14:00.000Z",
               votes: 0,
               article_id: 5,
+              comment_count: expect.any(String),
```

Comment:
I would like to steer you away from amending any previously passing tests to account for this. We introduced `expect.objectContaining` to account for the existence of any additional properties in future without causing previous tests to fail.
I would suggest to revert this test to how it was and to create a new test asserting only for the presence of the `comment_count`.
Also, we should be able to assert exactly how many comments this article has